.goreleaser.yml

```diff
@@ -15,7 +15,7 @@ builds:
     hooks:
       post:
         - cmd: bash -c "rm -f /tmp/packer-plugins-check;
-              ( go install github.com/hashicorp/packer/cmd/packer-plugins-check@latest && packer-plugins-check -load={{ .ProjectName }}_v{{ .Version }}_{{ .Env.API_VERSION }}_{{ .Os }}_{{ .Arch }} ) &>/tmp/packer-plugins-check && exit $? ||
+              ( go install github.com/hashicorp/packer/cmd/packer-plugins-check@v1.7.4 && packer-plugins-check -load={{ .ProjectName }}_v{{ .Version }}_{{ .Env.API_VERSION }}_{{ .Os }}_{{ .Arch }} ) &>/tmp/packer-plugins-check && exit $? ||
```

Comment:
Packer v1.7.6 was released today to address this issue. You should be able to revert back to using latest. Please let us know if you are still running into issues with the latest release. Cheers!